agent-app-go/internal/llm/ollama.go

```
@@ -64,6 +66,11 @@ func (c *ChatClient) Complete(ctx context.Context, system, user string) (string,
 	if err != nil {
 		return "", err
 	}
+	if f, ferr := os.OpenFile("/tmp/go-wire-requests.jsonl", os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0644); ferr == nil {
+		f.Write(body)
```

golangci-lint (2.12.2)

[error] Error return value of `f.Write` is not checked

(errcheck)